README.md

```diff
@@ -1,8 +1,59 @@
-# vscode-aks-tools
+# Azure Kubernetes Service (AKS) Extension for Visual Studio Code (Preview)
 
-* View your AKS clusters in the Kubernetes extension cloud explorer
-* Add AKS clusters to kubeconfig via the GUI
-* Run AppLens detector diagnostics on your AKS clusters
+The Visual Studio Code extension for Azure Kubernetes Service helps you view your AKS cluster, merge and save kubeconfig, perform AKS Diagnostics checks, and run AKS Periscope within VS Code.
+
+## Prerequisites
+
+Ensure that you have installed the [Kubernetes Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension for VS Code prior to installing the Azure Kubernetes Service Extension for Visual Studio Code following these [instructions](https://code.visualstudio.com/docs/azure/kubernetes#_install-the-kubernetes-extension).
+
+## Installation
+
+1. Download and install the [Azure Kubernetes Service extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) for Visual Studio Code.
+
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted.
+
+3. Once the installation is complete, you'll see a section named **Azure** under **Clouds**.
+
+4. Sign in to your Azure Account by clicking **Sign in to Azure…**
+
+## Features
+
+### Merge and Save Kubeconfig
```

Comment:
💡 idea share: `merge` and `save kubeconfig` appear as 2 options on `right click`, should be divide them into 2 different sub topics, along with screenshots? (Since we are detailing in-depth)

![image](https://user-images.githubusercontent.com/6233295/92339088-6bde8900-f108-11ea-8a79-d23281ff4363.png)

Reply:
I'm fine with the approach of dividing them into 2 sub topics though I imagine the description of each will be very short. Do you have any additional details you'd like me to add for each feature?

Reply:
If we are adding a screenshot of the right click menu, I think it would be better to keep one top-level screenshot of it in the beginning rather than having it in all sections unless "Merge into Kubeconfig" and "Save Kubeconfig" have additional GUI like the web view of AKS Diagnostics and AKS Periscope.

Reply:
I think covering these under one topic is fine.